packages/ra-core/src/controller/field/getResourceLinkPath.ts

```diff
@@ -62,7 +62,9 @@ const getResourceLinkPath = ({
         );
     }
     const sourceId = get(record, source);
-    const rootPath = basePath.replace(resource, reference);
+    const rootPath = basePath
```

Comment:
this is a change I wanted to push in the previous PR, but @djhi was too fast to merge it

Reply:
It was RFR! :joy: